frontend/src/components/StudentProfile/GraduationStatus/CertificationDogwoods.vue

```diff
@@ -59,12 +59,13 @@ import { useStudentStore } from "../../../store/modules/student";
 import { useSnackbarStore } from "@/store/modules/snackbar";
 import { base64ToFileTypeAndOpenWindow } from "../../../utils/common.js";
 import SchoolService from "@/services/SchoolService.js";
+import { useAppStore } from "../../../store/modules/app.js";
 
 export default {
   name: "CertificationDogwoods",
   created() {
     this.base64ToFileTypeAndOpenWindow = base64ToFileTypeAndOpenWindow;
-    this.isCertificateEligible();
+    this.isCertificateEligible(this.studentGradStatus.schoolOfRecordId);
```

Comment:
This line is no longer needed and can be removed. this.isCertificateEligible() was called on created() before because the function also fetched the school info from our APIs. Now we grab it from the store in the template so this is redundant.